File: lista 15questoes/15 questoes em python/att08.py
class Empregado:
    def __init__(self, nome, cargo, salario):
        self.nome = nome
        self.cargo = cargo
        self.salario = salario

class Empresa:
    def __init__(self, nome):
        self.nome = nome
        self.empregados = []

empresa1 = Empresa("Empresa X")
empregado1 = Empregado("Ruan", "Analista", 5000)
empresa1.empregados.append(empregado1)

File: lista 15questoes/15questoes em golang/att08.go
type Empregado struct {
    Nome   string
    Cargo  string
    Salario float64
}

type Empresa struct {
    Nome      string
    Empregados []Empregado
}

empresa1 := Empresa{Nome: "Empresa X"}
empregado1 := Empregado{Nome: "Ruan", Cargo: "Analista", Salario: 5000}
empresa1.Empregados = append(empresa1.Empregados, empregado1)
